pip/req/req_install.py

```diff
@@ -778,6 +779,14 @@ def install(self, install_options, global_options=(), root=None):
             self.install_succeeded = True
             return
 
+        # Extend the list of global and install options passed on to
+        # the setup.py call with the ones from the requirements file.
+        # Options specified in requirements file override those
+        # specified on the command line, since the last option given
+        # to setup.py is the one that is used.
```

Comment:
I recall this from the first go round I think  :  )
although, I can't find an example why this won't work, it's odd to see the resulting command using '-v'